doc/src/out_dir.md

```diff
@@ -0,0 +1,24 @@
+One common request is to have LALRPOP generate its files into Cargo's
+**output directory**, rather than storing them 'in-place' within the
+module tree. This can be done through the following configuration:
```

Comment:
I know we plan to make this the default. I figured I'd write up some instructions before hand.